src/AutoSysimages.jl

```diff
@@ -389,36 +392,44 @@ end
 This install the `asysimg` scripts.
 (Currently implemented only for Linux.)
 """
-function install(dir=_default_install_dir())
+function install(dir = _default_install_dir())
+
+    dir_exists = ispath(dir)
+    dir_exists || mkpath(dir)
+
     if Sys.islinux() || Sys.isapple()
```

Comment:
What about this?
``` julia
   if isnothing(dir)
       @warn """AutoSysimages: Installation is not yet supported for your OS.
Feel free to submit a PR."""
        return nothing
   end
```
... create dir ...
``` julia 
    script_fn = Sys.iswindows() ? "asysimg.bat" : "asysimg"
    script = joinpath(@__DIR__, "..", "scripts", "linux", script_fn)
```

Reply:
Ahhh, "linux".

Reply:
there is also the "linux" directory involved in the function call

Reply:
also, it would be more apropriate to have a `unix` directory because they both work on `linux` and `macos`

Reply:
I propose to rename the folder in a separate PR. Thus, I'll merge this because it looks good.